docs/src/examples/components/Label/Variations/index.tsx

```diff
@@ -29,6 +29,11 @@ const Variations = () => (
       description="The Icon component inside the Label can be defined with customizing it's prop"
       examplePath="components/Label/Variations/LabelExampleIconAsShorthand"
     />
+    <ComponentExample
+      title="Image"
```

Comment:
Image is part of the `Content` that a Label can contain.  This example should be moved from `Variations` to `Content`.